docs/extensions/network.md

```diff
@@ -0,0 +1,81 @@
+# Gardener Network Extension
+
+Gardener is an open-source project that provides a nested user model. Basically, there are two types of services provided by Gardener to its users:
+
+- Managed: users only request a Kubernetes cluster (Clusters-as-a-Service)
+- Hosted: users utilize Gardener to provide their own managed version of Kubernetes (Cluster-Provisioner-as-a-service)
```

Comment:
It's quite confusing what a "user" is. Can you clarify by using "users" for end-users (people only using shoot clusters but not operating/setting up Gardener itself) and "operators" for the other group?